src/main/scala/abalone/variant/Abalone.scala

```diff
@@ -20,10 +20,11 @@ case object Abalone
 
   override def baseVariant: Boolean = true
 
-  // TODO: Abalone set
-  override def initialFen = format.FEN("")
+  // pieces, turn, scoreP1, scoreP2, halfMovesSinceLastCapture (when = 100? => draw), total moves
+  override def initialFen = format.FEN("pp1PP/pppPPP/1pp1pp1/8/9/8/1PP1pp1/PPPppp/PP1pp 0 0 b 0 0")
```

Comment:
based on the comment, should the initial fen be:
`pp1PP/pppPPP/1pp1pp1/8/9/8/1PP1pp1/PPPppp/PP1pp b 0 0 0 0`?

Reply:
I was not sure as here https://www.chess.com/terms/fen-chess I see the turn is declared just after the pieces in second position but usually you decided to have the turn after the score if I remember well.
To keep consistency within this codebase I think it should then be `pp1PP/pppPPP/1pp1pp1/8/9/8/1PP1pp1/PPPppp/PP1pp 0 0 b 0 0` ?